spec/models/mentor_spec.rb

```diff
@@ -43,6 +43,20 @@
     end
   end
 
+  context "with scopes" do
+    describe "#order_by_full_name" do
+      it "returns the mentors ordered by full name" do
+        mentor1 = create(:mentor, first_name: "Anne", last_name: "Doe")
+        mentor2 = create(:mentor, first_name: "Anne", last_name: "Smith")
+        mentor3 = create(:mentor, first_name: "John", last_name: "Doe")
+
+        expect(described_class.order_by_full_name).to eq(
+          [mentor1, mentor2, mentor3],
+        )
+      end
+    end
```

Comment:
To ensure that this ordering is explicit, can we switch the order of definition compared to assertion please?

```suggestion
      it "returns the mentors ordered by full name" do
        mentor1 = create(:mentor, first_name: "Anne", last_name: "Smith")
        mentor2 = create(:mentor, first_name: "Anne", last_name: "Doe")
        mentor3 = create(:mentor, first_name: "John", last_name: "Doe")

        expect(described_class.order_by_full_name).to eq(
          [mentor2, mentor1, mentor3],
        )
      end
    end
```

This removes any ambiguity that it's just returning in the order it was defined.